toolbox/check_certificates.py

```diff
@@ -0,0 +1,400 @@
+#!/opt/privacyidea/bin/python
+
+
+__doc__ = """
```

Comment:
Please remove the `__doc__` part. It is not necessary. The docstring will be available via the `.__doc__` attribute of the module.